en/guides/manage-devices/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Introduction
+table_of_contents: true
+---
+
+# Introduction
+
+On some systems it is appropriate to have a system user. For example, with a system user, one can ssh to the device and run commands.
+
+By default on first boot, stock Core images run a program called console-conf. If the user has a keyboard and monitor attached to the device, they can use console-conf to configure the network and add a user account. This user account can then be used to ssh to the device for various management purposes.
```

Comment:
A couple of comments.  First, nice work, this really does a great job of documenting what a system-user is, and how to create one.

My first critique is that the first paragraph introduces the term system user, then the second paragraph talks about a user account.  In both cases, I think they mean the same thing, so maybe use system user in the second paragraph too?

The other point to note about a system user created via console conf, is that the user is automagically created using the store account's email address as the username, and that a password isn't set, but instead the SSH keys from the store account are provisioned.